requirements-dev.txt

```diff
@@ -3,3 +3,5 @@ django-debug-toolbar==1.8
 django-webtest==1.7.9
 factory-boy==2.8.1
 nplusone==0.8.1
+pdbpp==0.9.1
+ipython==6.2.1
```

Comment:
Do we need these in dev requirements?

Reply:
Nope!  I find them incredibly useful for helping with debugging and development, but if you rather I keep it local, happy to take them back out.

Reply:
I find them useful too, but probably best to keep them local so as not to annoy folks who have a different system. Plus fewer dependencies = easier ATO.

Reply:
Out they go!